shared/middleware/helpers/table.rb

```diff
@@ -140,12 +232,47 @@ def initialize(channel_id, storage_id, table_name)
     @table_name = table_name
 
     @hash = "#{@channel_id}:#{@table_name}:#{@storage_id}"
+    @metadata_hash = "#{@channel_id}:#{@table_name}:#{@storage_id}:metadata"
   end
```

Comment:
It's possible that I can. I think originally I was maybe trying to have the metadata live in the same table, and thus needed a separate hash. I can explore this further.